src/app/BUILD.gn

```diff
@@ -202,6 +202,7 @@ static_library("app") {
   public_deps = [
     ":app_config",
     "${chip_root}/src/access",
+    "${chip_root}/src/app/icd/:server-srcs",
```

Comment:
remove extra '/'.

Should this be conditional on ICD enabling?

Reply:
done